### DISCUSSION TITLE
Relaunch of osu! is required to join chat channels

### DISCUSSION THREAD

**Discussion by Rekunan** [ppy/osu#16391] [Q&A] [answered]
When I click on a chat channel to join, I have to relaunch osu! to actually join it.

Before relaunch after clicking:
[runtime.log](https://github.com/ppy/osu/files/7836725/runtime.log)
[performance.log](https://github.com/ppy/osu/files/7836726/performance.log)
[updater.log](https://github.com/ppy/osu/files/7836727/updater.log)
[network.log](https://github.com/ppy/osu/files/7836728/network.log)

After relaunch:
[runtime.log](https://github.com/ppy/osu/files/7836732/runtime.log)
[performance.log](https://github.com/ppy/osu/files/7836733/performance.log)
[updater.log](https://github.com/ppy/osu/files/7836734/updater.log)
[network.log](https://github.com/ppy/osu/files/7836735/network.log)

**Comment by smoogipoo** [comment, marked as answer]
I don't know what this means, because it seems to be working fine here. There's also nothing in the logs as far as I can see. Can you explain it via a video?

**Reply by Rekunan** [reply]
the video blacks out after i exit osu!, during which i click osu! on my taskbar to open it up again

https://user-images.githubusercontent.com/24501632/148718353-dbe43cee-ed5a-438f-966e-89b9cf4e8a9b.mp4

**Reply by smoogipoo** [reply]
Is it at the end of the list? If you scroll down the dropdown?

**Reply by Rekunan** [reply]
if you mean the dropdown in `...` then yes

**Reply by peppy** [reply]
Sounds like it's working as intended, then. The interface for channel display will be improved over time.
